packages/modules/pricing/src/migrations/.snapshot-medusa-pricing.json

```diff
@@ -440,26 +561,26 @@
           "nullable": false,
           "mappedType": "text"
         },
-        "name": {
-          "name": "name",
+        "attribute": {
+          "name": "attribute",
           "type": "text",
           "unsigned": false,
           "autoincrement": false,
           "primary": false,
           "nullable": false,
           "mappedType": "text"
         },
-        "rule_attribute": {
-          "name": "rule_attribute",
+        "value": {
+          "name": "value",
           "type": "text",
           "unsigned": false,
           "autoincrement": false,
           "primary": false,
           "nullable": false,
           "mappedType": "text"
         },
-        "default_priority": {
-          "name": "default_priority",
+        "priority": {
```

Comment:
q: Is this still present?

Reply:
Yeah it is, I am not sure if it's used though (at least it wasn't in the calculate prices logic). Should I leave it for now or remove it, WDYT?

Reply:
Just took a look, this was meant to act as a specificity override. i.e. if you wanted a particular price to show up instead of the opinionated one that pricing module spits out by default. 

This looks like a missed feature. I'd suggest leaving this for now and create a ticket to see if we want to even tackle this.